go/src/koding/klientctl/open/util.go

```diff
@@ -0,0 +1,61 @@
+package open
+
+import (
+	"os"
+	"path/filepath"
+)
+
+// Prepare a slice of paths to be absolute and/or in a format
+// that is friendly to the Koding IDE.
+//
+// Returns the same list as in, for an easy API.
+func PreparePaths(paths []string) ([]string, error) {
+	for i, path := range paths {
+		path, err := filepath.Abs(path)
+		if err != nil {
+			return nil, err
+		}
+		paths[i] = path
+	}
+	return paths, nil
+}
+
+// Like MkdirAll, Mkfiles creates any files as needed.
+func Mkfiles(paths []string, perm os.FileMode) (err error) {
+	for _, path := range paths {
+		if _, err := os.Stat(path); err != nil {
+			if os.IsNotExist(err) {
+				os.MkdirAll(filepath.Dir(path), perm)
+				// The file doesn't exist, create it.
+				f, err := os.Create(path)
+				if err != nil {
+					return err
+				}
+				f.Chmod(perm)
+				f.Close()
+			} else {
+				// If we don't know what to do with the error, it might be
+				// permissions/etc. Return it.
+				return err
+			}
```

Comment:
@rjeczalik code style fixed btw, if you'd like to change the function let me know, otherwise this is good to merge i believe.